tests/cpp/fwklib/UDPIpc.hpp

```diff
@@ -160,7 +160,11 @@ class UDPMessageQueues : public SharedTaskObject {
   std::string m_label;
 
  public:
-  explicit UDPMessageQueues(std::string label) : m_label(label) {}
+  explicit UDPMessageQueues(std::string label) : m_label(label) {
+    m_cntInbound = 0;
```

Comment:
Can we be consistent and either use the initializer style of constructor body but not both.

Reply:
Sure, sorry.